benchmark/string_concat.yml

```diff
@@ -1,6 +1,8 @@
 prelude: |
   CHUNK = "a" * 64
   UCHUNK = "é" * 32
+  SHORT = GC::INTERNAL_CONSTANTS[:BASE_SLOT_SIZE] / 2
+  LONG = GC::INTERNAL_CONSTANTS[:BASE_SLOT_SIZE] * 2
```

Comment:
These seem to be numbers (`SHORT == 20` and `LONG == 80`). Are they supposed to be integers or should they be strings?